tests/test_observability.py

```
@@ -0,0 +1,159 @@
+
+    def test_returns_false_on_exception(self):
+        """Should return False when urlopen raises any exception."""
+        with patch("urllib.request.urlopen", side_effect=OSError("refused")):
+            result = check_langfuse_service_available("http://localhost:3000")
+        assert result is False
+
+
+class TestInitializeLangfuseSentinel:
+    """Tests for the three-state init sentinel."""
+
+    def _reset_module_state(self):
```

Ruff (0.15.2)

[warning] Missing return type annotation for private function `_reset_module_state`

Add return type annotation: `None`

(ANN202)